src/App.tsx

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import SignInForm from "./pages/SignInForm/SigninForm";
+import DieryHome from "./pages/DiaryHome/DiaryHome";
+import {BrowserRouter, Route, Routes} from "react-router-dom";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+        <BrowserRouter>
```

Comment:
Use a separate route file to handling routings

Reply:
resolved in the Handling Issues(https://github.com/fcodelabs/training-web/pull/251) PR